common/reportMetrics.py

```diff
@@ -369,108 +373,343 @@ def process_replay(file_path):
+        line = metric_file.readline()
+        while line:
+            # Read each line from csv and generate a json
+            current_row = dict()
+            if current_line_count == parameters['chunkLines']:
+                send_data([to_send_metric_data, min_timestamp_epoch, max_timestamp_epoch], metric_file_path, chunk_count + 1)
+                to_send_metric_data = []
+                current_line_count = 0
+                chunk_count += 1
+
+            row = line.split()
+            if row[2:] == field_names[2:] or 'RESTART' in row or not row:
+                line = metric_file.readline()
+                continue
+
+            # make timestamp
+            time = row[0]
+            ampm = row[1]
+            if time == 'Average:':
+                line = metric_file.readline()
+                continue
+            timestamp = _get_timestamp_sar(date, time, ampm, parameters['timeZone'])
+            current_row['timestamp'] = str(timestamp)
+            if min_timestamp_epoch == 0 or min_timestamp_epoch > long(timestamp):
+                min_timestamp_epoch = long(timestamp)
+            if max_timestamp_epoch == 0 or max_timestamp_epoch < long(timestamp):
+                max_timestamp_epoch = long(timestamp)
+
+            # metric values
+            for i in range(2, len(row)):
+                column_name = field_names[i].strip()
+                metric_key = column_name
+                column_name = column_name + '[' + instance + ']:' + str(i)
+                # Generate normalization id or use from config.ini
+                if len(normalization_ids_map) != 0:
+                    if metric_key in normalization_ids_map:
+                        group_id = int(normalization_ids_map[metric_key])
+                else:
+                    group_id = column_name.split(":")[1]
+                column_name = column_name.split(":")[0] + ":" + str(group_id)
+                current_row[column_name] = row[i]
+            to_send_metric_data.append(current_row)
+            current_line_count += 1
+            line = metric_file.readline()
+        # send final chunk
+        if len(to_send_metric_data) != 0:
+            send_data([to_send_metric_data, min_timestamp_epoch, max_timestamp_epoch], metric_file_path, chunk_count + 1)
+            chunk_count += 1
+        logger.debug("Total chunks created: " + str(chunk_count))
+        save_grouping(grouping_map)
+    return
+
+
+def replay_db2(log_file_path):
+    print('Replaying db2 file')
+    with open(log_file_path) as log_file:
+        line_count = 0
+        chunk_count = 0
+        current_row = []
+        current_obj = dict()
+        start_time = time.time()
+        line = log_file.readline()
+        while line:
+            # empty line denotes new log entry. aggregate current string
+            if not line.strip():
+                # build json entry
+                entry = dict()
+                entry['tag'] = 'db2-replay'
```

Comment:
is a hostname or localhost can be used

Reply:
Will now use hostname in log entry or localhost if unavailable